Kudu.ContainerServices.Agent/Kudu.ContainerServices.Agent.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+	<PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <RuntimeIdentifiers>win10-x64</RuntimeIdentifiers>
```

Comment:
Is that what we used for existing projects?

Reply:
Yes, Joaquin showed me where several builds throughout dwas use the win10-x64 identifier and that is what we should use here.